docs/content/zh/docs/examples/containerd-e2e.md

```
@@ -0,0 +1,60 @@
+通过以下方式专注于单个工作负载：
+
+```bash
+sudo ptcpdump -i any --container-id <id> 'host 1.1.1.1'
+sudo ptcpdump -i any --container-name <name> 'host 1.1.1.1'
+```
+
+这些步骤复制了自动化测试中的覆盖范围，并可作为调试基于 containerd 的集群时的参考。
+
+## 输出示例
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)